### PR TITLE
Update OnMove event support

### PR DESCRIPTION
**Pull request by Kvaz1r**
Fix: #29

### CODE REVIEW

**Comment by sodevel** [comment]
Aren't Move-Events only generated for Toplevel-Windows? At least in the documentation of the latest sources i can find only this information, what makes you think that these events also work for every other wxWindow?

**Comment by Kvaz1r** [comment]
It [was fixed](https://github.com/wxWidgets/wxWidgets/commit/7a1bc568e5cb8d04dead3a1f53c3563ce41d4fef) in the documentation recently, but it also relates for 3.0 branch as well.

**Comment by sodevel** [comment]
Ah thanks, looks like my 1.5 week old build is 1 week too old :smile:

But anyway i see another issue, you added a code template only for C++, this should cause errors already in the preview for the other languages. Can you add the required templates also for the other languages?

**Comment by Kvaz1r** [comment]
At least I will try  ;-) Never used other languages so didn't know about it.

File: output/xml/default.xml [hunk 1]
```diff
@@ -179,6 +179,7 @@
 		<event name="OnPaint" class="wxPaintEvent" help="Process a wxEVT_PAINT event." />
 		<event name="OnSize" class="wxSizeEvent" help="Process a wxEVT_SIZE event." />
 		<event name="OnUpdateUI" class="wxUpdateUIEvent" help="Process a wxEVT_UPDATE_UI event." />
+		<event name="OnMove" class="wxMoveEvent" help="Process a wxEVT_MOVE event, which is generated when a window is moved. Not generated in wxGTK" />
```
Comment (sodevel) [inline output/xml/default.xml]: wxFB displays the events in the order they appear in the xml file, OnSize and OnMove are kind of related and now separated by OnUpdateUI. Can you place it between OnSize and OnUpdateUI instead please and also update the order in the code templates accordingly?

File: output/xml/default.luacode [hunk 2]
```diff
@@ -151,6 +151,9 @@
 
 		<!-- wxUpdateUIEvent -->
 		<template name="evt_connect_OnUpdateUI">#utbl$name:Connect( wx.wxEVT_UPDATE_UI, function(event)#nl --implements  #handler#nl #skip #nl end  )</template>
+		
+		<!-- wxSizeEvent -->
```
Comment (sodevel) [inline output/xml/default.luacode]: Copy-Paste error

File: output/xml/default.xml (hunk 1 above)
Comment (Kvaz1r) [reply output/xml/default.xml]: Fixed

**Pull request merged**